### ISSUE TITLE
Atea and nvALT

### ISSUE THREAD

**Issue by samuelcotterall** [pkamenarsky/atea#15]
I’d like to use Atea with [nvALT](http://brettterpstra.com/project/nvalt/), synced with Simplenote. I configured this accordingly: 

`{:file "/Users/samuelcotterall/Library/Application Support/Notational Data/todo.txt"}`

Atea picks up the changes made in nvALT, but I noticed that the UI doesn’t respond to click events (start Time tracking, quit, for example). This isn’t a problem when I point Atea to a file on my desktop. 

I could, I’m sure, move my nvALT notes out of `/Library/`, but I was wondering if this is a fixable issue.

**Comment by pkamenarsky** [comment]
Sounds like a bug - could you post a minimal example of a todo.txt that triggers that behavior?

**Comment by samuelcotterall** [comment]
Sure thing. Here’s the contents of the file https://gist.github.com/1879697: 

```
Order headphones
Return t-shirt
[Wedding] Call printer
```

And here’s a screenshot: http://cl.ly/3Z31104610102v320F35

Let me know if you need any more information.

**Comment by pkamenarsky** [comment]
That's strange - the todo file works for me (I can start / stop time tracking etc). Also there is no problem when I place it in an arbitrary folder in Application Support. The only thing I can  think of is that particular folder has some kind of special permissions so that the *csv file can't be created...

**Comment by samuelcotterall** [comment]
The CSV gets created without any problem (sorry, I should have mentioned that). 

I will have a play around with it tomorrow and see if I can pinpoint what’s causing the problems; I guess there are a lot of variables here.

**Comment by pkamenarsky** [comment]
That'd be great - I just can't reproduce the problem locally. Also maybe try to quit nVALT and see if it works then?

**Comment by samuelcotterall** [comment]
I’m sorry; this was my fault. 

At one point I accidentally renamed the file to `todo.txt.txt` (hidden extensions in Finder). Fixing this filename (and confirming in Terminal) meant that it updated the Atea menu item correctly, but failed when I interacted with it. This threw me off. 

I just trashed the file, started again, and it worked fine. Something must have gotten mixed up along the way.
